### PR TITLE
Added damping and user load function in Spring Mass example

### LINKED ISSUE #314: extend cases in springmass problem
- [x] add viscoity
 - [x] add user load
 - [x] add initial condition

### LINKED ISSUE #472: add working userloadfunc use example
check how is the user load func used in the static von mises problem

### PR DESCRIPTION
**Pull request by mvanzulli**
closes #314
closes #472

### CODE REVIEW

**Comment by jorgepz** [comment]
Could you add here the plot of the example? Just to see how it matches the solution? Thanks!

**Comment by mvanzulli** [comment]
> Could you add here the plot of the example? Just to see how it matches the solution? Thanks!

It happens something curious if I declare the boundaryConds(2) as empty (line 139 of `spirngMass.m` then It don't matches the analytic solution:

bug

**Comment by jorgepz** [comment]
Raro.... debería dar exactamente igual... sugiero q mires las matrices de masa... deberían ser iguales...

**Comment by jorgepz** [comment]
I would increase the final time... And if necessary reduce the deltat. And see what do we get in the plot....

File: examples/springMass/springMass.m [hunk 1]
```diff
@@ -139,14 +150,15 @@
 %md
 %md Execute ONSAS and save results: 
 [matUsHHT, loadFactorsMat] = ONSAS( materials, elements, boundaryConds, initialConds, mesh, analysisSettings, otherParams ) ;
+% stop%md
```
Comment (jorgepz) [inline examples/springMass/springMass.m:153]:
```suggestion
```

**Review by jorgepz — APPROVED** [review]

**Comment by mvanzulli** [comment]
> I would increase the final time... And if necessary reduce the deltat. And see what do we get in the plot....

This are the results:
Results

It seems that  if BC is declared as an empty struct then the error increases with time :-1:

**Comment by jorgepz** [comment]
seems to be working...
image
will merge.

**Comment by jorgepz** [comment]
i will do some tests... to see whats going on here! 😟

**Comment by mvanzulli** [comment]
> seems to be working... image will merge.

with an empty  field in the BC struct ? The results are obtained if the line 139 is uncomment

**Pull request merged**